containers/build-or-download-cvmfs-debs.sh

```diff
@@ -1,10 +1,12 @@
 #/bin/bash
 cvmfsversion=$1
-arch=$(dpkg --print-architecture)
-os=debian11
 
 apt-get update
-apt-get install -y wget
+apt-get install -y wget lsb-release
+
+arch=$(dpkg --print-architecture)
+os="$(lsb_release -si | tr [:upper:] [:lower:])$(lsb_release -sr)"
```

Comment:
```suggestion
os="$(lsb_release -si | tr [:upper:] [:lower:])$(lsb_release -sr | cut -d '.' -f 1)"
```
Not sure if this is the problem...